src/status_im2/contexts/syncing/setup_syncing/view.cljs

```diff
@@ -129,7 +120,16 @@
                :type            :grey
                :container-style {:margin-top 12}
                :icon-left       :i/copy}
-              (i18n/label :t/copy-qr)]])]]
+              (i18n/label :t/copy-qr)]])
+          (when-not (sync-utils/valid-connection-string? @code)
+            [rn/view {:style style/standard-auth}
+             [standard-auth/view
+              {:size                  :size/s-40
+               :track-text            (i18n/label :t/slide-to-reveal-code)
+               :customization-color   customization-color
+               :on-enter-password     on-enter-password
+               :biometric-auth?       false
```

Comment:
Quick question: Do we support biometric authentication?

Reply:
not yet, but we will! :)